thumbnails/mad4jekyll

```diff
@@ -0,0 +1 @@
+![mad4jekyll](https://cloud.githubusercontent.com/assets/15498648/11429252/f2ddb7b0-947d-11e5-9259-3efd315f2f6f.png)
```

Comment:
This is not a correct thumbnail. Thumbnail should be in image format (i.e jpg or png) with dimensions 250 x 200.